scrapli_community/nokia/sros/nokia_sros.py

```diff
@@ -61,7 +61,7 @@
     ),
     "configuration": (
         PrivilegeLevel(
-            pattern=r"^\*?[abcd]:[\w]+>config#\s?$",
+            pattern=r"\*?[abcd]:[\w\s-]+>config[\w>]*(#|\$)\s?$",
```

Comment:
is there any way to make sure this starts with a newline anchor (`^`)? that is kinda the "standard" and almost is necessary because I think we check if the pattern starts and ends w/ the line anchors and if so we add the multiline flag.

Reply:
Hey @carlmontanari,
Yes sure, as a matter of fact it's an issue of copy/paste as the newline anchor is there in the code I use. My bad...